Index: python13.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Search ,Sort , Search Sorted , Filter\r\nimport numpy as np\r\n\r\n#for checking condition or number index number\r\nvar = np.array([1,2,3,4,5,6])\r\nfinding_from_arr = np.where(var%2 == 1)\r\n\r\n#Checking the number index and condition where given input send and checking side left to right & retun the index number\r\n#from left to right\r\ninsert_element_sort = np.searchsorted(var,100)\r\ninsert_element_sort_r = np.searchsorted(var,100,side = 'right') #from right to left\r\n\r\nprint(finding_from_arr)\r\nprint(insert_element_sort_r)\r\nprint(insert_element_sort)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/python13.py b/python13.py
--- a/python13.py	(revision fec7aa709456e6741263d084612384fda826211a)
+++ b/python13.py	(date 1773850023854)
@@ -5,7 +5,8 @@
 var = np.array([1,2,3,4,5,6])
 finding_from_arr = np.where(var%2 == 1)
 
-#Checking the number index and condition where given input send and checking side left to right & retun the index number
+#Checking the number index and condition where given input send and
+# checking side left to right & retun the index number
 #from left to right
 insert_element_sort = np.searchsorted(var,100)
 insert_element_sort_r = np.searchsorted(var,100,side = 'right') #from right to left
